### ISSUE TITLE
[CI] Registry URL validation doesn't match template guidance

### ISSUE THREAD

**Issue by jaydeluca** [open-telemetry/opentelemetry.io#5511]
**What happened?**

When adding a new registry entry, and following [the template](https://github.com/open-telemetry/opentelemetry.io/blob/main/templates/registry-entry.yml), the comments seems to indicate that the `repo` input is the only mandatory URL:

```
urls:
  repo: https://github.com/your-organization/your-repo # (mandatory) projects don't have to be hosted on GitHub, but this should link to the git or other source control repository for your project
  docs: https://your-organization/your-instructions # (optional) path to a documentation for the specific project.
```

However, if you only specify the repo, you get the following errors when trying to build and run the project:

```
ERROR Website URL is missing for integrations registry entry 'JDBI'
ERROR Docs URL is missing for integrations registry entry 'JDBI'
```

**What did you expect would happen?**

Should the template be updated to indicate that all three fields are mandatory? Or should we update [this validation](https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/shortcodes/ecosystem/integrations-table.md?plain=1#L27-L32) to allow them to be left blank?

**Comment by vitorvasc** [comment]
Also referencing #8566 here, since both issues aim to update the template for [`registry-entry.yml`](https://github.com/open-telemetry/opentelemetry.io/blob/main/templates/registry-entry.yml?rgh-link-date=2025-12-03T09%3A31%3A53.000Z) and/or the workflow rules that validate it.

**Comment by svrnm** [comment]
yes, let's make sure the template is in sync with the validation!

**Comment by pmady** [comment]
Here is the PR https://github.com/open-telemetry/opentelemetry.io/pull/8592
